eternalcore-core/src/main/java/com/eternalcode/core/feature/teleport/TeleportServiceImpl.java

```diff
@@ -43,12 +42,11 @@ public void teleport(Player player, Location location) {
 
     @Override
     public Optional<Location> getLastLocation(UUID player) {
-        return Optional.ofNullable(this.lastPosition.get(player)).map(PositionAdapter::convert);
+        return Optional.ofNullable(this.lastPosition.get(player)).map(position -> PositionAdapter.convert(position));
```

Comment:
This change is needed?

Reply:
I did it because we agreed that most people don't understand how method reference works instead of Lambda, and we already use this convention in most places.